scripts/publish_to_bintray.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python3
```

Comment:
maybe rename this file, it is not very generalised so "upload_kiwix_lib_android_to_bintray" might help understanding

Reply:
Done